<launch>
   <node pkg="ctrl_pkg" name="sail_controller" type="sail_node.py">
      <rosparam command="load" file="$(find rotary_sensor_spi)/config/as5048_node.yaml"/>
   </node>
</launch>